manifests/Rustlang/rustup/1.21.1.yaml

```diff
@@ -0,0 +1,19 @@
+Id: Rustlang.rustup
+Publisher: Rustlang
```

Comment:
Not sure if this should just be `Rust`. The name of the language is just `Rust`, but it sometimes is called Rust-Lang (and various other forms there of) to disambiguate. The main confusion comes from the video game [Rust](https://en.wikipedia.org/wiki/Rust_(video_game))

Reply:
That's a good point. Considering that the format is [`manifests/publisher/application/version.yaml`](https://docs.microsoft.com/en-us/windows/package-manager/package/repository#step-3-add-your-manifest-to-the-local-repository), any `winget` content related to said game would be listed by the name of its publisher, Facepunch Studios.

I'll change it to Rust.